src/components/pager/css/Pager.css

```diff
@@ -3,67 +3,84 @@
  * Data and UI separated into 2 parts: Layout and Color Themes
  */
 
-/***********************************************************/
+/**********************************************************   */
```

Comment:
Does it have to have the space between the `*` at the end? If so, it might be better to just remove this comment break if it isn't really needed.